pkg/bootstrap/bootstrap.go

```diff
@@ -62,8 +62,27 @@ type Options struct {
 	// Issue tracking sync periods per controller: https://github.com/reddit/achilles-sdk/issues/171
 	SyncPeriod time.Duration
 
-	// LeaderElection determines whether the controller should use leader election (a form of active-passive HA).
+	// Determines whether the controller should use leader election (a form of active-passive HA).
 	LeaderElection bool
+
+	// LeaderElectionID determines the name of the resource (Lease) that leader election
+	// will use for holding the leader lock.
+	LeaderElectionID string
+
+	// LeaderElectionNamespace determines the namespace in which the leader
+	// election resource (Lease) will be created.
+	LeaderElectionNamespace string
+
+	// The renew deadline for this leader election controller.
+	// Must be set to ensure the resource lock has an appropriate client timeout.
+	// If set too low, a single slow response from the API server can result
+	// in losing leadership.
+	// Note: This must be set lower than the LeaderElectionLeaseDuration.
```

Comment:
Can we add validation and fail if the `LeaderElectionLeaseDuration` > `LeaderElectionRenewDeadline`?

Reply:
yes, good idea.

Reply:
Talked offline, controller-runtime already fails with a descriptive error message if this isn't the case. No need to implement our own validation.